packages/core/src/payment-integration/default-payment-integration-service.ts

```diff
@@ -285,4 +287,12 @@ export default class DefaultPaymentIntegrationService implements PaymentIntegrat
     async validateCheckout(checkout?: Checkout, options?: RequestOptions): Promise<void> {
         await this._checkoutValidator.validate(checkout, options);
     }
+
+    async handle(errorOrId: Error | string, key?: string): Promise<PaymentAdditionalAction> {
```

Comment:
Need to think of a name for this function that is more specific to the action the function performs. @bc-sebastianszafraniec what do you think?

Reply:
@bc-nick yes, maybe it is better to use some more specific name. What would you say for `handlePaymentHumanVerification`

Reply:
Much better for me than it is now